src/utils.ts

```diff
@@ -1,6 +1,7 @@
 import isEqual from 'lodash.isequal'
 import type { ParsedCSSValue } from 'webdriverio'
-import expect from 'expect'
+
+import { expect } from 'expect-webdriverio'
```

Comment:
Can you verify we don't have any circular dependencies?
```suggestion
import { expect } from './index.js'
```

Reply:
sure, fixed